From f8d7af68530d8a9aa0c972911d489fcd72a5ef74 Mon Sep 17 00:00:00 2001
From: Stef Walter <stefw@gnome.org>
Date: Mon, 6 Aug 2012 10:01:09 +0200
Subject: [PATCH] Make MDNS work in all but the most restrictive zones

 * MDNS is a discovery protocol, and much like DNS or DHCP should
   be available for the network to function as expected.
 * Avahi (the main MDNS) implementation has taken steps to make sure
   no private information is published by default.
 * See: https://fedoraproject.org/wiki/Desktop/Whiteboards/AvahiDefault
---
 config/zones/public.xml | 1 +
 config/zones/work.xml   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/config/zones/public.xml b/config/zones/public.xml
index 49fc4c20af52..95433962c259 100644
--- a/config/zones/public.xml
+++ b/config/zones/public.xml
@@ -3,6 +3,7 @@
   <short>Public</short>
   <description>For use in public areas. You do not trust the other computers on networks to not harm your computer. Only selected incoming connections are accepted.</description>
   <service name="ssh"/>
+  <service name="mdns"/>
   <service name="dhcpv6-client"/>
   <forward/>
 </zone>
diff --git a/config/zones/work.xml b/config/zones/work.xml
index f1a14a9b4682..eaf509023137 100644
--- a/config/zones/work.xml
+++ b/config/zones/work.xml
@@ -3,6 +3,7 @@
   <short>Work</short>
   <description>For use in work areas. You mostly trust the other computers on networks to not harm your computer. Only selected incoming connections are accepted.</description>
   <service name="ssh"/>
+  <service name="mdns"/>
   <service name="dhcpv6-client"/>
   <forward/>
 </zone>
-- 
2.31.1

